<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QMainWindow" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>672</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1777215</width>
    <height>1777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OneDayAway</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>671</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="GUI">
     <attribute name="title">
      <string>Visual</string>
     </attribute>
     <widget class="QLabel" name="inventoryLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Carrying:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="guiOutput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>375</y>
        <width>551</width>
        <height>91</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QLabel" name="itemsLabel">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>330</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>You can see:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gameContols">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>921</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="dropItem">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>10</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="southButton">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>110</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="westButton">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>60</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="northButton">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="takeItem">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>10</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="infoButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="mapButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="eastButton">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>60</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSlider" name="weight">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>20</width>
         <height>101</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="loudness">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>20</width>
         <height>101</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="weightLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Weight</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="loudnessLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Loudness</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="soundDisabled">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Sound Disabled</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="fullScreen">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Full Screen</string>
       </property>
      </widget>
      <widget class="QPushButton" name="resetGame">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="inventory">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>375</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="itemsInRoom">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>375</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="currentRoom">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>350</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Room</string>
      </property>
     </widget>
     <widget class="QLabel" name="gameImage">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>777</width>
        <height>333</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="textBased">
     <attribute name="title">
      <string>Text Based</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="welcomeText">
        <property name="text">
         <string>Welcome Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="output">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextEdit" name="textBasedOutput">
        <property name="font">
         <font>
          <family>Consolas</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="input">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="QLabel" name="enter">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;↵&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="userInput"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
